.github/workflows/integration-test-cassandra-suite.yaml

```diff
@@ -57,8 +57,9 @@ jobs:
         path: /home/runner/work/rook/rook/tests/integration/_output/tests/
 
     - name: setup tmate session for debugging
```

Comment:
The cassandra tests are already failing in other branches from #8483 so let's not block this PR for that.